Ascend.java

```diff
@@ -1,7 +1,20 @@
 // Generates three integer random numbers in a given range,

```

Comment:
OVERALL: 
1. This is a good work, but I would highly recommend sticking to what you were taught in class. Since you don't know what you will be given in exam.. Please stick to what we taught in recitation & lectures, please follow the guidelines.. 
2. There are some styling issues. see my comments. 
3. There are some logical issues in TimeFormat.java